internal/cli/iam/users/invite.go

```diff
@@ -246,12 +247,11 @@ func InviteBuilder() *cobra.Command {
 		Short: "Invite a user.",
 		Args:  require.NoArgs,
 		PreRunE: func(cmd *cobra.Command, args []string) error {
-			opts.OutWriter = cmd.OutOrStdout()
-			opts.InitInput(cmd.InOrStdin())
```

Comment:
the panic was happening because of this line? (which does not call the func returned)

Reply:
yes